src/vs/workbench/contrib/positronNotebook/browser/KernelStatusBadge.tsx

```diff
@@ -72,7 +110,7 @@ export function KernelStatusBadge() {
 		>
 			<div className='positron-notebook-kernel-status-badge' data-testid='notebook-kernel-status'>
 				<RuntimeStatusIcon status={runtimeStatus} />
```

Comment:
I noticed this doesn't rotate for non-idle states. Could add 
```tsx
			style={{
				color: colorCss,
				animation: status !== RuntimeStatus.Idle ? 'spin 2s linear infinite' : undefined
			}}
```
to the icon component to do that. (Or add to it's empty css file.)

Reply:
I think not rotating was a design choice for the console, which I've copied here